.github/workflows/translation_check.py

```diff
@@ -148,7 +150,7 @@ def count_translations(dir_with_translations, print_info, reference_file):
             print(f'\t{', '.join(messages["missing"])}')
             print(f'Unnecessary (not in en-GB) ({not_needed[lang]}):')
             print(f'\t{', '.join(messages["unnecessary"])}')
-            print(f'Unexpected (should not be translated) ({len(messages["unexpected"])}):')
+            print(f'Warning: Supossed to remain intact ({len(messages["unexpected"])})\nPlease review and if OK, add to supress_warning.py:')
```

Comment:
```suggestion
            print(f'Translation not expected ({len(messages["unexpected"])})\nPlease review and if OK, add to supress_warning.py:')
```

Not 100% satisfied with neither wording yet.